### PR TITLE
Revise explanation for kube-scheduler

### PR DESCRIPTION
**Pull request by VishalPraneeth**
Here https://kubernetes.io/docs/concepts/overview/components/#kube-scheduler
The first line is a bit confusing "Control plane component that watches for newly created Pods with no assigned nodes,...."
I think it could be "Control plane component that watches for newly created Pods that have not yet been assigned to any nodes" just for sake of simplicity.

This docs is modified based on the issue https://github.com/kubernetes/website/issues/39562 .
The file modified :
https://github.com/kubernetes/website/blob/5f8fdc6abb0fdcfa3853fbe97560def89aac032c/content/en/docs/reference/glossary/kube-scheduler.md

### CODE REVIEW

File: content/en/docs/reference/glossary/kube-scheduler.md [hunk 1]
```diff
@@ -4,15 +4,15 @@ id: kube-scheduler
 date: 2018-04-12
 full_link: /docs/reference/command-line-tools-reference/kube-scheduler/
 short_description: >
-  Control plane component that watches for newly created pods with no assigned node, and selects a node for them to run on.
+  Control plane component that watches for newly created Pods that have not yet been assigned to any nodes, and selects a node for them to run on.
```
Comment (NitishKumar06) [inline content/en/docs/reference/glossary/kube-scheduler.md:7]: Hi, thanks for raising this PR. I'd suggest you go through the [Documentation Style Guide](https://kubernetes.io/docs/contribute/style/style-guide/) to understand the style for writing Kubernetes docs.

File: content/en/docs/reference/glossary/kube-scheduler.md [hunk 2]
```diff
@@ -4,15 +4,15 @@ id: kube-scheduler
...
 
 aka: 
 tags:
 - architecture
 ---
 Control plane component that watches for newly created
-{{< glossary_tooltip term_id="pod" text="Pods" >}} with no assigned
-{{< glossary_tooltip term_id="node" text="node">}}, and selects a node for them
+{{< glossary_tooltip term_id="pod" text="Pods" >}} that have not yet been assigned to any
+{{< glossary_tooltip term_id="node" text="nodes">}}, and selects a node for them
```
Comment (NitishKumar06) [inline content/en/docs/reference/glossary/kube-scheduler.md:15]: The text is not as confusing as it sounds like if ready carefully. If we go by this mean, I think we'll come across multiple such texts for which no issue/PR has ever been raised (unless that's critical).

**Review by NitishKumar06 — COMMENTED** [review]
The issue has the label `needs-triage`. Please go through [this](https://www.kubernetes.dev/docs/guide/issue-triage/).

**Comment by VishalPraneeth** [comment]
> The issue has the label `needs-triage`. Please go through [this](https://www.kubernetes.dev/docs/guide/issue-triage/).

Yes sure, I will go through the label👍.

**Pull request closed without merging**